scripts/arangodb-docker-container-setup/docker-container-scripts/test-data/testData.json

```diff
@@ -366,9 +366,7 @@
                     }
                 },
                 "username": "cool-james",
-                "roles": [
```

Comment:
These appear to be formatting differences. Can you diagnose why this changed? Please revert this file if merging in.